kernel-util/src/main/java/org/sakaiproject/content/util/ZipContentUtil.java

```diff
@@ -394,8 +401,36 @@ private void storeContentCollection(String rootId, ContentCollection collection,
 	 * @param out
 	 * @throws Exception
 	 */
-	private void storeContentResource(String rootId, ContentResource resource, ZipOutputStream out) throws Exception {		
-		String filename = resource.getId().substring(rootId.length(),resource.getId().length());				
+	private void storeContentResource(String rootId, ContentResource resource, ZipOutputStream out) throws Exception {
```

Comment:
Isn't this the same code that is used for compressing folders from the resources tool as well? If you have a folder/file that has the same ID as a user then it will get a strange name in the export.